am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestDataFactory.java

```diff
@@ -34,15 +37,22 @@ public static ExplicitAccessGrant createGrantForWholeDocument(String resourceId,
     public static ExplicitAccessGrant createGrantForWholeDocument(String resourceId,
                                                                   String accessorId,
                                                                   Set<Permission> permissions) {
+        return createGrant(resourceId, Stream.of(accessorId).collect(Collectors.toSet()),
```

Comment:
No `Stream.of` please. Use guava lib.

Reply:
Delegate to `createGrantForWholeDocument(String resourceId, Set<String> accessorId, Set<Permission> permission)`

Reply:
👍